docs/FAQs.md

```diff
@@ -11,3 +11,11 @@ As this is an evolving product, this requirement may be adjusted in the future,
 - Test Pilot is about building a community through FxA enabled forums such as Discourse.
 - We also use the account as a gate since we are doing certain metrics collections that we want to ensure people have knowingly elected to opt into.
 
+
+### Why does Test Pilot require Javascript?
+
+- We use Javascript to render the user interface of Test Pilot.
+- We use Javascript to manage experiment installations.
+- We use Javascript to collect data to help us improve Test Pilot in accordnace with our [Terms of Use](https://testpilot.firefox.com/terms) and [Privacy Notice](https://testpilot.firefox.com/privacy).
+
+Since Test Pilot is an open source project, you can see all of the cool ways we use Javascript by [examining our code](https://github.com/mozilla/testpilot/).
```

Comment:
Nit: technically it's "JavaScript" (upper S)
Source: Us. We invented it. https://developer.mozilla.org/en-US/docs/Web/JavaScript